### ISSUE TITLE
doc: Specify Ubuntu Desktop Edition Preferred

### ISSUE THREAD

**Issue by aiyer-commits** [RobotLocomotion/drake#13210]
Hi Team,

I've been experimenting with running drake in Virtualbox, running Ubuntu 18.04 Server Edition (which does not come with X11 installed), and only realized that XOrg/X11 is needed while running the "rendering multi-body plant" tutorial via jupyter notebook. I realized this thanks to vtk crashing my jupyter notebook kernel, as seen in this screenshot:

![Screen Shot 2020-05-02 at 6 55 32 PM](https://user-images.githubusercontent.com/56838551/80894146-9924b180-8ca6-11ea-9fa9-7ca7eb36188a.png)

I am working on configuring my ubuntu server edition vm to use X11, will report back on if it can be done.

**Comment by aiyer-commits** [comment]
Ok I got it to work in Ubuntu Server Edition by installing xorg and xvfb, following the instructions here (https://docs.pyvista.org/getting-started/installation.html). 

Would it make sense to add docs specific to Ubuntu Server Edition?

**Comment by jwnimmer-tri** [comment]
Yes, when listing Drake's dependencies it probably makes sense to note that any rendering-related features require an X server.

**Comment by jamiesnape** [comment]
Alluding to the title, I don't think we (should) prefer either. CI uses "minimal" FWIW.

**Comment by jwnimmer-tri** [comment]
Good point. I'd meant to update the title during triage, but forgot.
